homeassistant/components/media_source/__init__.py

```diff
@@ -37,6 +37,9 @@ async def async_setup(hass: HomeAssistant, config: dict):
     hass.data[DOMAIN] = {}
     hass.components.websocket_api.async_register_command(websocket_browse_media)
     hass.components.websocket_api.async_register_command(websocket_resolve_media)
+    hass.components.frontend.async_register_built_in_panel(
+        "media-browser", "Media Browser", "hass:play"
```

Comment:
```suggestion
        "media-browser", "Media Browser", "hass:folder-multiple-image"
```

Reply:
![image](https://user-images.githubusercontent.com/18730868/92256776-c4f4b580-ee99-11ea-9943-bddd2c08d1f8.png)

Reply:
I chose play without really searching :)

Reply:
No worries, I updated your suggestion as well to have the second param changed too so I could do it in one commit.

Reply:
Looks like just images now maybe?
Maybe `play-box-multiple`?
![image](https://user-images.githubusercontent.com/5662298/92263029-43724700-eedc-11ea-8fc0-0de7ef9e09cd.png)